Code/Tools/SceneAPI/SceneCore/Utilities/DebugOutput.cpp

```diff
@@ -241,7 +246,12 @@ namespace AZ::SceneAPI::Utilities
             }
             dbgFile.Close();
 
+            // XML is useful because it stores more information than JSON with the serializer, so some automation is better suited to use XML.
```

Comment:
Did we consider making format to write a setting? Do most use cases want both JSON and XML representations?

Reply:
these files only get generated when you run Asset Processor with --debugOutput, and I figure it's better to just output them all instead of having different commands per file, --debugOutputXML, --debugOutputJSON, --debugOutputHumanReadable.

Reply:
Could easily have --debugOutput --debugFormat=All|XML|Json etc